var/spack/repos/builtin/packages/awscli/package.py

```diff
@@ -11,8 +11,7 @@ class Awscli(PythonPackage):
     """This package provides a unified command line interface to
        Amazon Web Services"""
 
-    homepage = "https://pypi.org/project/awscli/"
-    url      = "https://pypi.io/packages/source/a/awscli/awscli-1.16.308.tar.gz"
+    pypi = "awscli/awscli-1.16.308.tar.gz"
```

Comment:
Was vaguely iffy about removing the `homepage`, but I think this result looks super readable.

Reply:
Yeah, as far as `spack info` is concerned, nothing has changed. It's just defined in the base class now.